k8s/iabweb/wagtail.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: wagtail
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: wagtail
+  template:
+    metadata:
+      labels:
+        app: wagtail
+    spec:
+      securityContext:
+        fsGroup: 1000
+        runAsNonRoot: true
+      containers:
+        # -----------------------------------------------------
+        # wagtail Container
+        # -----------------------------------------------------
+        - name: wagtail
+          image: "ghcr.io/ietf-tools/www:$APP_IMAGE_TAG"
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+              name: http
+              protocol: TCP
+          volumeMounts:
+            - name: iabwww-tmp
+              mountPath: /tmp
+            - name: iabwww-nginx
+              mountPath: /var/lib/nginx
+            - name: iabwww-media
+              mountPath: /app/media
+            - name: iabwww-cfg
+              mountPath: /app/supervisord.conf
+              subPath: supervisord.conf
+            - name: iabwww-cfg
+              mountPath: /app/ietf/settings/local.py
+              subPath: local.py
+            - name: iabwww-cfg
+              mountPath: /etc/nginx/nginx.conf
+              subPath: nginx.conf
+            - name: iabwww-cfg
+              mountPath: /etc/nginx/sites-enabled/default
+              subPath: nginx-default.conf
+          env:
+            - name: "CONTAINER_ROLE"
+              value: "iabweb"
+            - name: "DJANGO_SETTINGS_MODULE"
+              value: "ietf.settings.production"
+            # ensures the pod gets recreated on every deploy:
+            - name: "DEPLOY_UID"
+              value: "$DEPLOY_UID"
+          envFrom:
+            - configMapRef:
+                name: iabwww-django-config
+            - secretRef:
+                name: iabwww-secrets-env
```

Comment:
Don't you need a `secrete-env` defined in this dir?

Reply:
Similar to https://github.com/ietf-tools/datatracker/blob/main/k8s/secrets.yaml

Reply:
I'm happy to add `secrets.yaml` with a placeholder for each required variable in the secret.

On a side note, please note that we have both a config map and an external secret here. We may choose to consolidate all variables into the latter in the future, if that's preferred.

Reply:
Updated: https://github.com/ietf-tools/www/pull/477/commits/61b2de443b6d7df39db377a98cfe990d1ff08992